bin/container.sh

```diff
@@ -164,31 +149,31 @@ start_wasabi() {
   wenv="WASABI_CONFIGURATION=-DnodeHosts=${wcip} -Ddatabase.url.host=${wmip}"
 
 #   fixme: try to reuse the start_container() method instead of 'docker run...' directly; currently a problem with quotes in ${wenv} being passed into container.
-  docker run --net=${docker_network} --name ${project}-main -p 8080:8080 -p 8090:8090 -p 8180:8180 -e "${wenv}" -d ${project}-main
-
-  if [[ "${waittime}" == "ping" ]]; then
-    echo -ne "${green}chill'ax ${reset}"
-    for trial in $(seq 1 20); do
-      curl ${mip}:8080/api/v1/ping >/dev/null 2>&1
-      status=$?
-      [[ ${status} -eq 0 ]] && break
-#      beerMe
-      echo -ne "${green}\xF0\x9F\x8D\xBA ${reset}"
-      sleep 3
-    done
-    [[ ${status} -ne 0 ]] && usage "\nGiving up on the ping. Wasabi might not be running!" 1
-  else
-    beerMe "${waittime}"
-  fi
+  docker run --net=${docker_network} --name ${project}-main -p 8080:8080 -p 8090:8090 -p 8180:8180 \
+    -e "${wenv}" -d ${project}-main || \
+    usage "docker run --net=${docker_network} --name ${project}-main -p 8080:8080 -p 8090:8090 -p 8180:8180 -e \"${wenv}\" -d ${project}-main" 1
+
+  echo -ne "${green}chill'ax ${reset}"
+
+  status=0
+
+  for trial in $(seq 1 20); do
+    wget -q http://localhost:8080/api/v1/ping
```

Comment:
We can change this for loop to something like `wget -q --spider --tries=20 --waitretry=3 http://localhost:8080/api/v1/ping` - the spider option also avoids the download.